sdk/cosmos/azure-cosmos-dotnet-benchmark/src/main/java/com/azure/cosmos/dotnet/benchmark/ParallelExecutionStrategy.java

```diff
@@ -85,6 +85,10 @@ public RunSummary execute(
 
                 Summary currentTotalSummary = new Summary();
                 for (IExecutor executor : executors) {
```

Comment:
isn't it possible to avoid adding a null executor to executors collection in earlier stage when populating the executors?

Reply:
It is only null when the array item for the executor hasn't been initialized - basically a race between the initialization and the timer firing the first time